apps/desktop-wallet/src/api/apiDataHooks/wallet/useFetchWalletBalancesTokensBy.ts

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useQueries, UseQueryResult } from '@tanstack/react-query'
+
+import { combineBalancesByAddress, combineBalancesByToken } from '@/api/apiDataHooks/wallet/combineBalances'
+import useFetchWalletLastTransactionHashes from '@/api/apiDataHooks/wallet/useFetchWalletLastTransactionHashes'
+import { addressTokensBalancesQuery, AddressTokensBalancesQueryFnData } from '@/api/queries/addressQueries'
+import { useAppSelector } from '@/hooks/redux'
+
+export const useFetchWalletTokensBalancesByToken = () => useFetchWalletBalancesTokensBy(combineBalancesByToken)
+
+export const useFetchWalletTokensBalancesByAddress = () => useFetchWalletBalancesTokensBy(combineBalancesByAddress)
```

Comment:
This file exports 2 hooks. I think I prefer this approach over:
1. Creating a separate file for `useFetchWalletBalancesTokensBy`
2. Separate file for the simple `useFetchWalletTokensBalancesByToken`
3. Separate file for the simple `useFetchWalletTokensBalancesByAddress`

Reply:
Fine by me!

Reply:
It's correct actually, see latest PR. Moving Tokens/Balances at the end makes for nice sorting of files:

<img width="393" alt="image" src="https://github.com/user-attachments/assets/5632ce71-3c03-40fc-9c2e-929774129b49">

Reply:
Ok but then, should the hook names be updated?

Reply:
They should. I'm sorry that I didn't push it in this branch..! But it's in the next one 😇